managers/asset-operations-manager.js

```diff
@@ -149,10 +149,7 @@ class AssetOperationsManager {
             );
         }
 
-        return {
-            transactionHash: receipt.transactionHash,
-            status: receipt.status,
-        };
+        return receipt;
```

Comment:
We shouldn't change the way returned object looks like, it would be a breaking change otherwise. I suggest we just add additional field `operation`